pkg/cmd/server/api/types.go

```diff
@@ -83,6 +95,8 @@ const (
 	ControllersAll = "*"
 )
 
+type FeatureSet map[OpenShiftFeature]util.Empty
```

Comment:
> So I can't write something like this to update master the disabled
> features:

At runtime, duplicates don't matter.  I also don't object to you have an `AddAll` or `AddUnique`, but stable ordering for user provided values is valuable.